tests/subspace_search/test_cca.py

```diff
@@ -0,0 +1,47 @@
+import pytest
+
+import numpy as np
+import torch
+
+from latte.models.cca import estimation as cca_estimation
+from latte.evaluation import subspace_evaluation
+from latte.manifolds import utils as manifold_utils
+
+
+rng = np.random.default_rng(42)
+torch.manual_seed(42)
+
+
+@pytest.mark.parametrize(
+    "N",
+    [16000],
```

Comment:
Can we make this smaller? Currently the tests are running for 1h and it may be either a technical glitch or a problem with our unit tests...

Reply:
We can talk about this tomorrow :)